app/controllers/v3/packages_controller.rb

```diff
@@ -165,7 +165,9 @@ def create_fresh
       unprocessable_non_bits_package!
     end
 
-    PackageCreate.create(message:, user_audit_info:)
+    package = PackageCreate.create(message:, user_audit_info:)
```

Comment:
I would prefer to move this into `PackageCreate.create` and `PackageCopy.copy` (after the transaction in which the new package is saved). And I would add an if-condition as well:
```
if package.state == PackageModel::READY_STATE
  BitsExpiration.new.expire_packages!(package.app)
end
```